packages/quantic/force-app/main/default/lwc/quanticSearchBox/quanticSearchBox.js

```diff
@@ -210,6 +210,13 @@ export default class QuanticSearchBox extends LightningElement {
     this.input.blur();
   }
 
+  handleKeyValues() {
+    if (this.searchBox.state.value !== this.input.value) {
+      this.suggestionList?.resetSelection();
+      this.searchBox.updateText(this.input.value);
+    }
+  }
```

Comment:
Should we actually have a debounce as well?
Like a small delay before calling the suggestions?

Reply:
I don't know if it s relevant here regarding how the component should work, but if so we need to do this improvement in several locations on this file